src/Controller/AppController.js

```diff
@@ -0,0 +1,70 @@
+import { gameMessage } from '../constants.js';
+import { Console } from '@woowacourse/mission-utils';
+import Parser from '../utils/Parser.js';
+import CarController from './CarController.js';
+import ViewOutput from '../ViewOutput.js';
+import Calculator from '../utils/Calculator.js';
+import Validator from '../utils/Validator.js';
+
+export default class AppController {
+
+    carList = [];
+
+    async control() {
+        const [carNameList, inputTime] = await this.userInput();
+        this.createCar(carNameList);
+        this.playRacingCar(inputTime);
+        this.selectWinners();
+    }
+
+    async userInput() {
+        const inputCarName = await Console.readLineAsync(gameMessage.inputCarNameMessage);
+        const carNameList = this.checkCarName(inputCarName);
+
+        const inputTime = await Console.readLineAsync(gameMessage.inputTimeMessage);
+        this.checkTime(inputTime);
+
+        return [carNameList, inputTime];
+    }
```

Comment:
배열로 여러 요소를 반환하면 각 속성에 대한 의미부여를 할 수 없어서 상대적으로 가독성과 유지보수성이 떨어진다고 생각해요.
이런 경우에 배열을 return 하는것 보단 key를 통해서 의미부여를 할 수 있는 객체를 반환하는게 어떨까 싶어요~!
```suggestion
async userInput() {
    const inputCarName = await Console.readLineAsync(gameMessage.inputCarNameMessage);
    const carNameList = this.checkCarName(inputCarName);

    const inputTime = await Console.readLineAsync(gameMessage.inputTimeMessage);
    this.checkTime(inputTime);

    return { carNameList, inputTime };
}
```

Reply:
저도 같은 의견입니다. 배열로 반환할 경우 index 에 의존해야 하는데 , index 특성상 배열이 어지러워질 경우 그 값을 보장할 수 없고 , 또한 사용하는 곳에서 이해하기 힘들 수 있다는 생각이 들었습니다.

Reply:
key로 반환할 생각은 못 했네요.
key로 반환하면 좀 더 가독성있고, 유지보수에 순서도 지키지 않아도 되어서 훨씬 편하겠군요!
감사합니다!